docs/integrations/vscode-extension/README.md

```diff
@@ -0,0 +1,52 @@
+# DevCycle Feature Flags extension for Visual Studio Code
```

Comment:
should these be ignored and pulled in when building the docs?

Reply:
sure, i could replace it with a shorter header like 'DevCycle VSCode Extension' maybe?

Reply:
Oh I meant the markdown files shouldn't be committed to the docs repo. Since they're pulled from the extension repo this could create confusion of someone tries to edit this file

Reply:
ohhh i see, yeah i just realized the generated CLI folder is gitignored - I'll make a new PR and add the vscode folder